.github/workflows/shellcheck.yml

```diff
@@ -36,4 +36,5 @@ jobs:
         run: $HOME/bin/shellcheck-v0.7.1/shellcheck e2e/**/*.*sh
       - name: Check release script
         run: $HOME/bin/shellcheck-v0.7.1/shellcheck scripts/release.sh
-
+      - name: Check update-frontend-canister.sh script
```

Comment:
See on:pull_request:paths section at top of file.  This workflow ran this time because we altered something under .github, but would not have run for a future PR that only changed scripts/update-frontend-canister.sh.